src/apphosting/backend.ts

```diff
@@ -0,0 +1,482 @@
+import * as clc from "colorette";
```

Comment:
Optional - this files has a mix of prompting functions and API calling functions. In the past, I've see this lead to really large and unwieldy files. If you'd like to get ahead of that, consider splitting this into  backendPrompts.ts and backend.ts